scripts/resolve_suggestions_urls.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import json
+import os
+from jsonpath_rw import parse
+from jsonpointer import set_pointer
+
+SUGGESTIONS_API_URL = (
+    "https://cdn.eq.census-gcp.onsdigital.uk/data/{version}/{region}/{language}"
+)
+VERSION = "v4.0.0"
+
+
+def json_path_to_json_pointer(json_path):
+    """
+    Convert a json path string into a json pointer string.
+    :param json_path: the input data to search
+    :return: json pointer equivalent to the json path
+    """
+    json_pointer = json_path.replace("[", "").replace("]", "").replace(".", "/")
+    return f"/{json_pointer}"
+
+
+def resolve_schema(schema_filepath, suggestions_api_url=None):
```

Comment:
Again, I think the suggestions_api_url arg could be named better. `suggestions_url_root` seems to make sense

Reply:
Changed, thanks!👍